config/environments/development.rb

```diff
@@ -13,30 +13,23 @@
   config.eager_load = false
 
   # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
+  config.consider_all_requests_local = true
   config.action_controller.perform_caching = false
 
-  # Don't care if the mailer can't send.  
-  # config.action_mailer.default_url_options = { :host => 'localhost:3000' }
-#<<<<<<< HEAD
+  # Don't care if the mailer can't send.
   config.active_support.deprecation = :notify
-  config.action_mailer.default_url_options = { :host => 'dev.fsektionen.se' }
+  config.action_mailer.default_url_options = {host: 'dev.fsektionen.se'}
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.perform_deliveries = true
   config.action_mailer.raise_delivery_errors = true
-  config.action_mailer.default :charset => "utf-8"
-  
+  config.action_mailer.default charset: "utf-8"
+
   config.action_mailer.smtp_settings = {
-  address: "localhost",
-  port: 1025
+      address: "localhost",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.